source/common/stats/isolated_store_impl.h

```diff
@@ -39,6 +39,15 @@ template <class Base> class IsolatedStatsCache {
     return *new_stat;
   }
 
+  const Base* get(StatName name) const {
```

Comment:
put this in private: section?

Reply:
I think it needs to be public--we're calling it on the IsolatedStatsCache instances in the IsolatedStoreImpl below, and they don't appear to be friends.